README.md

```diff
@@ -32,6 +32,12 @@ You should publish Honey's config file using the following [Artisan](https://lar
 php artisan vendor:publish --tag=honey
 ```
 
+Run database migrations (adds a spammers table):
```

Comment:
Could we update this to "Finally, you should run your database migrations, which adds the spammers table:"

Just so that it ties in nicely with the 'style' of the rest of the readme.